tests/js/common/shell/shell-view-arangosearch.js

```diff
@@ -43,6 +43,14 @@ var ERRORS = arangodb.errors;
 function ViewSuite () {
   'use strict';
   return {
+    tearDown : function () {
+      try {
+        db._dropView("abc");
+      } catch (_) {}
+      try {
```

Comment:
See above: Does this need try/catch?

Reply:
I think right now it does not throw, but try catch does not make it worse